2-mandatory/3-financial-times.js

```diff
@@ -23,59 +31,78 @@ function titleWithFewestWords(allArticleTitles) {
     (Hint: remember that you can also loop through the characters of a string if you need to)
 */
 function headlinesWithNumbers(allArticleTitles) {
-    // TODO
+  //means one or more digit
+  var r = /\d+/;
+  articleTitleWithNum = [];
```

Comment:
@lalehkoupa  please do not use regex for now,it is great to know but it is better to hard code it for now ;)

Reply:
Sure!